Magic/src/main/java/com/elmakers/mine/bukkit/action/builtin/CastAction.java

```diff
@@ -0,0 +1,76 @@
+package com.elmakers.mine.bukkit.action.builtin;
+
+import com.elmakers.mine.bukkit.api.spell.MageSpell;
+import org.bukkit.configuration.ConfigurationSection;
+
+import com.elmakers.mine.bukkit.action.BaseSpellAction;
+import com.elmakers.mine.bukkit.api.action.CastContext;
+import com.elmakers.mine.bukkit.api.magic.Mage;
+import com.elmakers.mine.bukkit.api.spell.SpellResult;
+
+import javax.annotation.Nonnull;
+
+public class CastAction extends BaseSpellAction {
```

Comment:
What is the intended use case for this action?

Reply:
People have been using the Command action in spells to cast other spells, which seemed silly.

Reply:
[My example](https://github.com/grisstyl/BetterPotter/blob/master/spells/item_abilities/trick_wand.yml) currently uses the CommandAction to cast a randomized, weaker version of a spell which seemed odd, as Nathan said.